src/components/calculator.js

```diff
@@ -1,62 +1,60 @@
 /* eslint-disable react/destructuring-assignment */
```

Comment:
- [ ] OPTIONAL: I would suggest you learn to use the destructuring assignment.  In React, we use it all the time.  

In fact, it seems that this `estlint-disable` is no longer needed, because you did use destructuring assignment, and upon deleting this line there were no eslint errors or warnings on my end.